pandas/guide/source/pandas_setup.rst

```diff
@@ -68,6 +70,8 @@ repository:
 * Install pandas development dependencies:
     ``conda install -c defaults -c conda-forge --file=<pandas-dir>/ci/requirements-optional-conda.txt``
 
+(For Window users: run the above commends in Anaconda Prompt)
```

Comment:
Personally I prefer to be more specific and not assume much prior knowledge from users, and I'd let them know where to find the Anaconda Prompt.

Also, if you find an easy way to rewrite this paragraph so this doesn't come at the end, that would be great. To avoid users following the above instructions in Windows, to realize when they reach this point that they were doing it the "wrong" way.